lib/encoder.py

```diff
@@ -766,7 +770,11 @@ def _FilesToEncodings(self, files, videofile, bitrate=None,
         encoder or self._FileNameToEncoder(full_filename),
         bitrate or _FileNameToBitrate(full_filename),
         videofile or _FileNameToVideofile(full_filename))
-      candidate.Recover()
+      try:
+        candidate.Recover()
+      except Error:
```

Comment:
#84 filed, TODO added. Filed all broken encodings in self.bad_encodings rather than printing to the console.

Added unittest for this behavior.